devops/docker/fastapi/Dockerfile

```diff
@@ -2,9 +2,11 @@ FROM continuumio/miniconda3
 
 WORKDIR /app/
 
+RUN apt-get update && apt-get install -y python3-opencv -y libgl1-mesa-dev -y xvfb
 RUN conda install -c conda-forge python=3.9 adios2 ffmpeg
-RUN pip install gunicorn
+RUN pip install gunicorn opencv-python-headless
```

Comment:
Why do we need opencv-python-headless, is `Xvfb` not enought?

Reply:
Yes! I missed this, thanks! Xvfb is enough. `opencv-python-headless` was my first attempt to get things working, no longer needed though.